src/content/docs/apis/nerdgraph/examples/nerdgraph-streaming-export.mdx

```diff
@@ -72,22 +72,60 @@ To set up the IAM role:
 
 When that's done, you can work on setting up your export rules using NerdGraph. For more on that, keep reading.
 
+
+## Set up an Azure Event Hub [#event-hub-setup]
+
+To set up streaming data export to Azure, you must first set up an Event Hub. We'll walk you through that procedure in the next three steps.
+
+Alternatively, you can follow the Azure guide [here](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-create).
+
+### Step 1. Create an Event Hubs Namespace [#create-event-hubs-namespace]
+
+Create a dedicated Firehose to stream your New Relic data to:
+
+1. Navigate to Event Hubs within your Microsoft Azure account.
+2. Follow the steps to create an Event Hubs Namespace. We recommend enabling 'Auto-Inflate' to ensure you receive all of your data.
+3. Ensure public access is enabled, we will be using a Shared Access Policy to authenticate securely with your Event Hub.
+4. Once your Event Hubs Namespace is deployed, click '**Go to resource**.'
+
+### Step 2. Create an Event Hub[#create-event-hub]
+
+1. In the left column, click on **Event Hubs**
+2. Then click **+Event Hub** to create an Event Hub
+2. Enter the desired Event Hub Name. Save this, as we will need it later to create the Streaming Export Rule
+4. Once the Event Hub is created, click on the Event Hub
+
+### Step 3. Create and Attach a Shared Access Policy [#event-hub-policy]
+
+1. In the left column, navigate to **Shared access policies**
+2. Click **+Add** near the top of the page
+3. Choose a name for your **Shared access policy**
+4. Check **Send**, and click create
+5. Click the created policy, and copy the **Connection string–primary key**. Save this, as New Relic will use this to authenticate and send data to your Event Hub.
```

Comment:
Small detail but other points in the numbered lists are ended with `.` so we might add them whenever they're missing to be consistent